src/System.Runtime/src/System.Runtime.csproj

```diff
@@ -38,6 +38,10 @@
     <Compile Include="System\Reflection\RuntimeReflectionExtensions.cs" />
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == '' OR '$(TargetGroup)' == 'netstandard1.7' OR '$(TargetGroup)' == 'uap101aot'">
```

Comment:
No idea... The first type is defined in here... is there a reference to S.R.Handles? 

https://github.com/dotnet/corefx/blob/bffef76f6af208e2042a2f27bc081ee908bb390b/src/System.Runtime.Handles/src/System/IO/HandleInheritability.cs